protocol/contracts/oud/OudRedeemer.sol

```diff
@@ -0,0 +1,145 @@
+pragma solidity ^0.8.17;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Oud Redeemer (protocol/contracts/oud/OudRedeemer.sol)
+
+import '@openzeppelin/contracts/access/Ownable.sol';
```

Comment:
Please use explicit imports (good practice / safety to limit the scope of what's pulled in)